index.html

```diff
@@ -1010,12 +1023,6 @@ <h3>Overview</h3>
         a combination of the two. Finally we process each <a>context</a> contained
         in the <a>local context</a> <a>array</a> as follows.</p>
 
-      <p class="changed">Unless specified using
-        <a data-link-for="JsonLdOptions">processingMode</a> API option,
-        the <a>processing mode</a> is set using the <code>@version</code> <a>entry</a>
-        in a local <a>context</a> and
-        affects the behavior of algorithms including <a>expansion</a> and <a>compaction</a>.</p>
-
```

Comment:
Note that what we've effectively done is have a 1.0 mode; setting `@version` to 1.1 only serves to stop things which have configured to run in 1.0 mode. All the places we say to update `processingMode` to `json-ld-1.1`, if necessary, don't actually have any effect.